drone_data_analyze.py

```diff
@@ -0,0 +1,10 @@
+from src.utils import DataAnalyzer
+from src.utils import Parser
+
+def main():
```

Comment:
Just a note for both of us: let's figure out how to move this file and `plot_localization_history` into a separate `scripts` folder or something so there are only the two main files in the parent directory